src/NServiceBus.Core.Tests/AssemblyScanner/When_checking_image_type.cs

```diff
@@ -1,29 +0,0 @@
-﻿namespace NServiceBus.Core.Tests.AssemblyScanner
-{
-    using System.IO;
-    using NUnit.Framework;
-
-    [TestFixture]
-    public class When_checking_image_type
```

Comment:
@timbussmann While we still had tests that looked for x64 or x86 assemblies, in the actual `AssemblyScanner` code, we were no longer looking at that. We were instead trying to load the assembly and letting it throw a BadImageFormatException if there is a mismatch. We were only using the `Image` code to skip non-assembly files.

That's why I didn't add anything to `AssemblyValidator` to look for that and removed the tests.